app/services/survey_monkey_parser.rb

```diff
@@ -43,15 +43,15 @@ def questions
   end
 
   class ResponseParser
-    ATTRIBUTES = %w[score comment response_id survey_id created_at language].freeze
+    ATTRIBUTES = %w(score comment response_id survey_id created_at collector_id custom_variables url language).freeze
 
     def initialize(data, survey)
       @data = data
```

Comment:
what about wrap data to open struct?
attr_accessor: data
`@data = OpenStruct.new(data)`

then we can do,
delegate :collector_id, :custom_variables, :analyze_url to: data

alias_method :analyze_url, :url

Then we can easy read from data. Just a propose.